packages/cache/src/cache.ts

```diff
@@ -86,23 +86,24 @@ export async function restoreCache(
   }
 
   const compressionMethod = await utils.getCompressionMethod()
+  let archivePath = ''
+  try {
```

Comment:
So this will essentially suppress all failures including temp directory creation and tar extraction failures. Essentially we are trading accuracy for continuity here. We definitely need a monitor which will tell us how many users are hitting such failures.

Reply:
This suppression is already there https://github.com/actions/cache/blob/c3f1317a9e7b1ef106c153ac8c0f00fed3ddbc0d/src/restore.ts#L55-L62  and https://github.com/actions/cache/blob/main/src/save.ts#L51-L59 , the only thing is now it is moved to toolkit instead of Github Actions.

Reply:
> We definitely need a monitor which will tell us how many users are hitting such failures.

I will check how we can monitors for this

Reply:
This is same problem as what @anuragc617 would be looking into w.r.t getting warnings from cache action so that we can have monitor on those.